packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js

```diff
@@ -22,11 +26,61 @@ const schemaDirName = 'schema';
 
 function failOnInvalidAuthType(usedDirectives, opts) {
   if (usedDirectives.includes('auth') && !opts.isUserPoolEnabled) {
-    throw new Error(`You are trying to use the @auth directive without enabling Amazon Cognito user pools for your API.
-Run \`amplify update api\` and choose "Amazon Cognito User Pool" as the authorization type for the API.`);
+    throw new Error(`You are trying to
+use the @auth directive without enabling Amazon Cognito user pools for your API.
+Run \`amplify update api\` and choose
+"Amazon Cognito User Pool" as the authorization type for the API.`);
```

Comment:
In the light of multi-auth being merged, we should rephrase this to 'enable' or 'add' Cognito, because this wording could suggest that it works if *only* Cognito enabled.